### ISSUE TITLE
GMC chatqna issue

### ISSUE THREAD

**Issue by lianhao** [opea-project/GenAIInfra#258]
Using the latest versions of retriever-usvc, we'll see curl failed with error code `52` with the following command:

```bash
kubectl -n test-app-chatqna exec client-test-7b7f97ddd9-fjdvv -- curl http://router-service:8080   -X POST  -d '{"text":"What is the revenue of Nike in 2023?","parameters":{"max_new_tokens":17, "do_sample": true}}' -H 'Content-Type: application/json'
```
Similar to something like [this](https://github.com/opea-project/GenAIInfra/actions/runs/10193145162/job/28197877549?pr=251)

When checking with the router pod log, it shows some errors when talking to llm service, complaining something about `payload is too big` and `timeout`. Switch the model from Intel/neural-chat-7b-v3-3 to bigscience/bloom-560m, we don't see this error any more. But this issue needs to be resolved any way.

**Comment by zhlsunshine** [comment]
Hi @lianhao, according to your command,  it seems that the router service has a problem, `http://router-service:8080`, does it miss the namespace information?  May it be `http://router-service.test-app-chatqna:8080`?

**Comment by lianhao** [comment]
@zhlsunshine It's not a DNS issue. K8S will automatically append the namespace when doing DNS name resolving.

**Comment by zhlsunshine** [comment]
Hi @lianhao, should this issue be closed?

**Comment by lianhao** [comment]
closed as PR #356 is merged
